src/storage-preview/azext_storage_preview/operations/account.py

```diff
@@ -103,7 +103,7 @@ def update_storage_account(cmd, instance, sku=None, tags=None, custom_domain=Non
     if custom_domain is not None:
         domain = CustomDomain(name=custom_domain)
         if use_subdomain is not None:
-            domain.name = use_subdomain == 'true'
+            domain.use_sub_domain = use_subdomain == 'true'
```

Comment:
Can you change this to `use_sub_domain_name` so that when new [sdk](https://github.com/Azure/azure-sdk-for-python/pull/3851/files#diff-405ddd0c1a1405b64a5627a8ad998583R41) is vendored, this does not need to be updated again.